glitter/templatetags/glitter.py

```diff
@@ -14,11 +27,14 @@ def glitter_head(context):
     with permission to edit the page.
     """
     user = context.get('user')
+    render_context, request = _get_context_request(context)
+    rendered = ''
 
     if user is not None and user.is_staff:
         template = get_template('glitter/include/head.html')
```

Comment:
Try:

```
template = context.template.engine.get_template('glitter/include/head.html')
rendered = template.render(context)
```

Then you shouldn't need `_get_context_request`.

You might need another solution for multiple templates with the function below though - that I'll leave to you to figure out!